docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+
+version: '3'
+
+services:
+  registry:
+    restart: always
+    image: registry:2
+    ports:
+      - 5000:5000
+    environment:
+      REGISTRY_HTTP_TLS_CERTIFICATE: /certs/domain.crt
+      REGISTRY_HTTP_TLS_KEY: /certs/domain.key
+      REGISTRY_AUTH: token
+      REGISTRY_AUTH_TOKEN_REALM: Registry Realm
+      REGISTRY_AUTH_TOKEN_SERVICE: Token services
+      REGISTRY_AUTH_TOKEN_ISSUER: Registry token issuer
+      REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE: /root/certs/bundle
+    volumes:
+      - /telescope/data:/var/lib/registry
```

Comment:
We should figure out where in the host we're going to put our stuff.

Reply:
I think `/usr/share/our_directory` would make sense.

Reply:
When you say `our_directory`, would that mean this directory,[ `/mnt/docker0storage/registry`](https://github.com/Seneca-CDOT/docker.cdot.systems/pull/2/files#diff-3d957ab66eeff42f24ce303126a7f8debe6c90ff0eb4674f68b8637b2d195218R12)?

Reply:
Correct, so it would be:

```
volumes:
  - /mnt/docker0storage/registry:/var/lib/registry
```

Such that the container will overlay `/mnt/docker0storage/registry` onto `/var/lib/registry` within the container.